### PR TITLE
ticket 9 ready for code review

### PR DESCRIPTION
**Pull request by KarlLuther**
Hello

Everything seems to work fine, although I am not sure that I have done enough error tests(but I couldn’t come up with anything else) 

Thank you
Ivan

### CODE REVIEW

File: controllers/deleteComment-controller.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { removeComment } = require("../models/removeComment-model");
+
+exports.deleteComment = (req, res, next) => {
+  const { params } = req;
+  const id = params.comment_id;
+  removeComment(id)
+    .then((rows) => {
+      res.status(204).send({});
```
Comment (callumu21) [inline controllers/deleteComment-controller.js:8]: We don't want to be sending anything back here, so either no argument in the .send() would work, or you can make use of res.sendStatus().

**Review by callumu21 — APPROVED** [review]
Nice work, Ivan! You just need to change your test and controller, so that the response sent back is completely empty rather than an empty object. Otherwise, this looks good to merge 👍
